docs/extensions.html

```diff
@@ -109,23 +89,30 @@ <h1 class="title">
 
 <section class="section">
   <div class="container">
-    {% for extension in page.extensions %}
-      <a class="box" href="{{ extension.url }}">
-        <div class="columns">
-          <div class="column is-one-third">
-            <h3 class="title is-4">
-              <strong>{{ extension.name }}</strong>
-            </h3>
+    <div class="columns is-multiline">
+      {% for extension in page.extensions %}
+      <div class="column is-3">
```

Comment:
This makes the layout very narrow. You can't really see what the extension is about. The image is the most important part and should be the bigger element here.

<img width="857" alt="screen shot 2017-11-30 at 18 19 40" src="https://user-images.githubusercontent.com/1254808/33447582-adad8172-d5fb-11e7-859b-eb3a8f77b5fe.png">